osu.Game.Tests/Visual/SongSelect/TestSceneBeatmapLeaderboard.cs

```diff
@@ -422,6 +416,8 @@ public void SetRetrievalState(PlaceholderState state)
             {
                 PlaceholderState = state;
             }
+
+            public void SetScores(ICollection<ScoreInfo> scores) => Scores = scores;
```

Comment:
I know it's test code but it's a bit of a weird one, can this one just be simple delegation to base? i.e.

```csharp
            public new ICollection<ScoreInfo> Scores
            {
                get => base.Scores;
                set => base.Scores = value;
            }
```

Reply:
sure?